Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN chmod 755 /bin/ttn-lw-stack
 COPY ttn-lw-cli /bin/ttn-lw-cli
 RUN chmod 755 /bin/ttn-lw-cli
 
+RUN mkdir -p /srv/ttn-lorawan/device-repository && /bin/ttn-lw-stack dr-db init
```

Comment:
I don't know if this is a good idea, as every (re-)build of the Docker image will result in a potentially different device repository.

If we go this way I think it would be better to have the device repository as a git submodule of the lorawan-stack repository, and build the index from that. That way we know exactly which commit of the device repository will end up in our images.

We'd do something similar for the frequency plans repository and webhook template repository (in a separate pull request).

Reply:
I would rather we avoid making significant changes to the repository and the release process in this particular PR. The Dockerfile is pretty much tied with our release workflow either way. In that sense, I think it is better to do something like this for now:

Add a CI step for fetching the device-repository archive from GitHub, and then use that for the Dockerfile, e.g.

```
ADD device-repository/ /tmp/device-repository
RUN mkdir -p /srv/ttn-lorawan/device-repository && /bin/ttn-lw-stack dr-db init --dr.config-source=directory --dr.directory=/tmp/device-repository
```

And create a separate issue for handling the frequency plans, webhook templates and the device repository repositories.

Reply:
Opened a PR: https://github.com/TheThingsNetwork/lorawan-stack/pull/3671

Reply:
Updated configuration and Dockerfile accordingly, assuming that `data/lorawan-devices` is a submodule for the Device Repository

Reply:
What is the verdict?

Reply:
@neoaggelos

Reply:
With https://github.com/TheThingsNetwork/lorawan-stack/pull/3671 merged, I will rebase this PR and update the Dockerfile